Configuration/StandardSequences/python/Validation_cff.py

```diff
@@ -48,7 +48,8 @@
 fastSim.toReplaceWith(prevalidation,_prevalidation_fastsim)
 
 from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
-phase2_common.toReplaceWith(prevalidation, prevalidation.copyAndExclude([cms.SequencePlaceholder("mix"),globalPrevalidation,metPreValidSeq,jetPreValidSeq]))
+phase2_common.toReplaceWith(prevalidation, prevalidation.copyAndExclude([cms.SequencePlaceholder("mix"),globalPrevalidation,metPreValidSeq]))
+phase2_common.toReplaceWith(jetPreValidSeq, hltJetPreValidSeq)
```

Comment:
see discussion in the mother PR https://github.com/cms-sw/cmssw/pull/48722

Reply:
This will disappear once the current branch is rebased on top of the latest changes on ```pre5```.